lib/collection/widgets/second_pass_food_item_card.dart

```diff
@@ -34,7 +34,6 @@ class SecondPassFoodItemCard extends StatelessWidget {
       DropdownMenuItem(child: Text('Wild food'), value: 'Wild food'),
       DropdownMenuItem(child: Text('Food aid'), value: 'Food aid'),
       DropdownMenuItem(child: Text('Other'), value: 'Other'),
```

Comment:
Might be worth adding TODOs to replace all the old dropdownmenu implementations if that is the plan

Reply:
Yep, I think this is basically Issue #15 — I made a comment about it there as I think the solution is to replace `DropdownMenuItems` by `DropdownSearch` — I think it now makes more sense to track it as an Issue as there are quite many places where `DropdownMenuItem` is used in the code.